### DISCUSSION TITLE
AttributeError: module 'supervision' has no attribute 'get_video_frames_generator'

### DISCUSSION THREAD

**Discussion by ArupSankarRoy** [roboflow/supervision#984] [Q&A]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar bug report.

### Bug

AttributeError                            Traceback (most recent call last)
[<ipython-input-6-1e7f30b0a37f>](https://localhost:8080/#) in <cell line: 2>()
      1 # extract video frame (get image from the video)
----> 2 generator = sv.get_video_frames_generator(VIDEO_PATH)
      3 iterator = iter(generator)
      4 frame = next(iterator)
      5 sv.plot_image(frame,(16,16))

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by SkalskiP** [comment]
What version of supervision are you using?

**Comment by ArupSankarRoy** [comment]
0.1.0

**Comment by onuralpszr** [comment]
> 0.1.0



~~0.1.0 doesn't have that feature. Please install the latest.~~


Update: I checked whether the feature is there or not. but since __init__.py wasn't configured probably as it is today you can't directly call via "sv. get_video_frames_generator("file.mp4")" you need to type the whole library path to call it. Using the latest would be a lot easier to get it done. 

Old version URL: https://github.com/roboflow/supervision/blob/0.1.0/supervision/video/source.py

**Comment by onuralpszr** [comment]
I am converting this to Q&A since it is not a bug.
